README.md

```diff
@@ -3,13 +3,44 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  Speed depends on many things besides the complexity of code; it is based on hardware, other processes running, and the size of the information. The size of information is represented by n inside of asymptotic analysis. The size of information Can really affect time. Even if you have the second smallest asymptotic analysis log n, It is still possible for an algorithm to take ridiculously long if it has a lot of information to process.
```

Comment:
But the input size is directly quantified by asymptotic analysis -- that larger inputs take longer directly follows from that. I don't see how this is misleading.